front/src/components/Dialogs/TaskImportDialog.vue

```diff
@@ -1,31 +1,53 @@
 <template>
-  <q-dialog ref="dialogRef" @hide="onDialogHide">
+  <q-dialog ref="dialogRef" no-backdrop-dismiss @hide="onDialogHide">
     <q-card class="ctfnote-dialog">
       <q-card-section class="row">
         <div class="text-h6">Import tasks</div>
         <q-space />
         <q-btn v-close-popup icon="close" flat round dense />
       </q-card-section>
+
       <q-card-section class="q-pa-none">
         <q-tab-panels v-model="tab" animated>
-          <q-tab-panel name="parse" class="full-width">
-            <div>
+          <q-tab-panel
+            name="parse"
+            class="full-width q-pt-none"
+            style="height: 401px"
+          >
+            <q-form class="q-gutter-sm">
```

Comment:
Are you sure this should be a `q-form`? this translates to a real `<form>` and that could be a source of refreshing bugs since in my previous experience sometimes browsers decide to refresh pages after form submission stuff or something like that, especially when there is no submit connected to it.